src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.ImportWorkloads.targets

```diff
@@ -42,14 +42,17 @@ Copyright (c) .NET Foundation. All rights reserved.
   </Target>
 
   <!--  This target is not part of the build. Only used by dotnet workload restore command. Global property "SkipResolvePackageAssets"
-        need to be set to "true" to avoid requiring restore (which would likely fail if the required workloads aren't already installed).-->
+        need to be set to "true" to avoid requiring restore (which would likely fail if the required workloads aren't already installed).
+        In addition, since this is a target that is called on potentially-unsupported project types like esproj, we need to not fail
+        if the Target is missing. -->
   <Target Name="_GetRequiredWorkloads" DependsOnTargets="GetSuggestedWorkloads;PrepareProjectReferences" Returns="@(_ResolvedSuggestedWorkload)">
     <MSBuild
       Projects="@(_MSBuildProjectReferenceExistent)"
       Targets="_GetRequiredWorkloads"
       BuildInParallel="$(BuildInParallel)"
       Condition="'%(_MSBuildProjectReferenceExistent.BuildReference)' == 'true' and '@(_MSBuildProjectReferenceExistent)' != ''"
-      RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)$(_GlobalPropertiesToRemoveFromProjectReferences);TargetFramework;TargetFrameworks">
+      RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)$(_GlobalPropertiesToRemoveFromProjectReferences);TargetFramework;TargetFrameworks"
+      SkipNonexistentProjects="true" >
```

Comment:
Why should this help if there's a referenced (and existing) project that doesn't have the Target? Wouldn't it still fail?

Reply:
Oh good grief I somehow have the correct version in my modified global SDK install and then use the wrong metadata when I changed the code in the SDK and did the PR. 

Sometimes I wonder how I even tie my shoes.

Reply:
Wait, you tie your shoes? I thought you work from home 😉